cognite/client/_api/assets.py

```diff
@@ -1061,6 +1082,42 @@ def list(
                 >>> client = CogniteClient()
                 >>> my_label_filter = LabelFilter(contains_all=["PUMP", "VERIFIED"])
                 >>> asset_list = client.assets.list(labels=my_label_filter)
+
+            Using advanced filter, find all assets that have a metadata key 'timezone' starting with 'Europe',
+            and sort by external id ascending:
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import filters as flt
+                >>> client = CogniteClient()
+                >>> in_timezone = flt.Prefix(["metadata", "timezone"], "Europe")
+                >>> res = client.assets.list(advanced_filter=in_timezone, sort=("external_id", "asc"))
+
+            Note that you can check the API documentation above to see which properties you can filter on
+            with which filters.
+
+            To make it easier to avoid spelling mistakes and easier to look up available properties
+            for filtering and sorting, you can also use the `AssetProperty` and `SortableAssetProperty` Enums.
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import filters as flt
+                >>> from cognite.client.data_classes.assets import AssetProperty, SortableAssetProperty
+                >>> client = CogniteClient()
+                >>> in_timezone = flt.Prefix(AssetProperty.metadata_key("timezone"), "Europe")
+                >>> res = client.assets.list(
+                ...     advanced_filter=in_timezone,
+                ...     sort=(SortableAssetProperty.external_id, "asc"))
+
+            Combine filter and advanced filter:
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import filters
+                >>> client = CogniteClient()
+                >>> not_instrument_lvl5 = filters.And(
+                ...    filters.ContainsAny(("labels"), ["Level5"]),
+                ...    filters.Not(filters.ContainsAny(("labels"), ["Instrument"]))
```

Comment:
Extra parens snuck in it seems:
```suggestion
                ...    filters.ContainsAny("labels", ["Level5"]),
                ...    filters.Not(filters.ContainsAny("labels", ["Instrument"]))
```

Reply:
I think I used them intentionally to be consistent with the api docs but parens in this case don't change the output:
```python
>>> f.ContainsAny(("labels"), ["blah", "blah2"]).dump()
{'containsAny': {'property': ['labels'], 'values': ['blah', 'blah2']}}
>>> f.ContainsAny("labels", ["blah", "blah2"]).dump()
{'containsAny': {'property': ['labels'], 'values': ['blah', 'blah2']}}
```

Reply:
fixed in all 4 apis